lib/yggdrasil/command/move.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.Command.MoveCommand do
+  defstruct exit: nil
+
+  defimpl Yggdrasil.Command, for: Yggdrasil.Command.MoveCommand do
+    import Yggdrasil.Room.Context
+
+    def execute(cmd, room_ctxt) do
+      room_ctxt
+      |> notify_player("You head towards #{cmd.exit}")
+    end
```

Comment:
Commands would now compose what they want to happen. They would also have the ability to check if the exit exists for example and produce proper messages to tell the player. Maybe have to also add a `:success` key to the context so if one thing fails the whole command could be ignored.... I don't know. Later concerns. We need so much other stuff in place.